sdk/computefleet/azure-resourcemanager-computefleet/src/test/java/com/azure/resourcemanager/computefleet/ComputeFleetManagerTests.java

```diff
@@ -0,0 +1,281 @@
+                                                                    .withIpConfigurations(
+                                                                        Arrays.asList(
+                                                                            new VirtualMachineScaleSetIPConfiguration()
+                                                                                .withName(vmName)
+                                                                                .withProperties(
+                                                                                    new VirtualMachineScaleSetIPConfigurationProperties()
+                                                                                        .withPrimary(true)
+                                                                                        .withSubnet(
+                                                                                            new ApiEntityReference()
+                                                                                                .withId(network.subnets().get("default").id())
+                                                                                        )
+                                                                                        .withLoadBalancerBackendAddressPools(
+                                                                                            loadBalancer.loadBalancingRules()
+                                                                                                .get(loadBalancerName + "-lbrule")
+                                                                                                .innerModel().backendAddressPools()
+                                                                                        )
+                                                                                )
+                                                                        )
+                                                                    )
+                                                            )
+                                                    )
+                                                )
+                                                .withNetworkApiVersion(NetworkApiVersion.fromString("2024-03-01"))
+                                        )
+                                )
+                                .withComputeApiVersion("2024-03-01")
+                                .withPlatformFaultDomainCount(1)
+                        )
+                        .withRegularPriorityProfile(new RegularPriorityProfile()
+                            .withAllocationStrategy(RegularPriorityAllocationStrategy.LOWEST_PRICE)
+                            .withMinCapacity(1)
+                            .withCapacity(2)
+                        )
+                )
+                .create();
+            // @embedStart
```

Comment:
typo? `@embedEnd`?

Reply:
Fixed in the new version.

Reply:
Wait, isn't it `@embedmeStart`? 
https://github.com/microsoft/typespec/blob/main/packages/http-client-java/generator/http-client-generator-mgmt/src/main/java/com/microsoft/typespec/http/client/generator/mgmt/model/projectmodel/CodeSample.java#L24-L25

Reply:
Change to `// @embedmeStart` and `// @embedmeEnd"`

Reply:
Thanks. There seem other places having similar issue,
 e.g. https://github.com/Azure/azure-sdk-for-java/blob/bef39e2a502d913442e2ba96429f9f0cd0b455f6/sdk/applicationinsights/azure-resourcemanager-applicationinsights/src/test/java/com/azure/resourcemanager/applicationinsights/ApplicationInsightsManagerTests.java#L86-L102

Please help fix them in new PR..

Reply:
Yeah, do a search and fix in a follow-up PR.

![image](https://github.com/user-attachments/assets/cbde2714-1c30-40a8-8c72-ab245e5921be)

Reply:
Ok, I will fix its by another PR.

Reply:
Fix them by [PR#42357](https://github.com/Azure/azure-sdk-for-java/pull/42357)